build_with_cmake/napi/hello.cc

```diff
@@ -0,0 +1,23 @@
+#include <assert.h>
+#include <node_api.h>
+
+napi_value Method(napi_env env, napi_callback_info info) {
+  napi_status status;
+  napi_value world;
+  status = napi_create_string_utf8(env, "Hello, world!", 13, &world);
+  assert(status == napi_ok);
+  return world;
+}
+
+#define DECLARE_NAPI_METHOD(name, func)                                        \
+  { name, 0, func, 0, 0, 0, napi_default, 0 }
+
+napi_value Init(napi_env env, napi_value exports) {
```

Comment:
This method should also be static.

Reply:
Fixed in https://github.com/nodejs/node-addon-examples/pull/114.